src/main/scala/org/ergoplatform/network/ModeFeature.scala

```diff
@@ -0,0 +1,73 @@
+package org.ergoplatform.network
+
+import com.google.common.primitives.Ints
+import org.ergoplatform.nodeView.state.StateType
+import org.ergoplatform.settings.NodeConfigurationSettings
+import scorex.core.network.PeerFeature
+import scorex.core.network.PeerFeature.Id
+import scorex.core.serialization.Serializer
+import scorex.core.utils
+
+import scala.util.Try
+
+case class ModeFeature(stateType: StateType,
+                       verifyingTransactions: Boolean,
+                       popowBootstrapping: Boolean,
+                       popowSuffix: Int,
+                       blocksToKeep: Int) extends PeerFeature {
+
+  override val featureId: Id = 16: Byte
```

Comment:
It might be better to separate global object with this constants, e.g `org.ergoplatform.settings.PeerFeatureIds`

Reply:
done